ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -0,0 +1,12 @@
+import ivy
+
+
+def cholesky(x, /, *, symmetrize_input=True):
```

Comment:
Since you're implementing a frontend function, the signature and behavior should be the same as JAX, I think the previous commit was fine and was ready to merge since it was passing the unit tests.

Reply:
Previous implementation was based on [jax.numpy.linalg.cholesky](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.cholesky.html) instead of [jax.lax.linalg.cholesky](https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.linalg.cholesky.html#jax.lax.linalg.cholesky). So, if it is okay for you, I can revert to the old one.

Reply:
sorry for the confusion, yes we should be using the lax implementation! you're on the right track